bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/GhostScienceAreaGeometry.scala

```diff
@@ -0,0 +1,19 @@
+package edu.gemini.spModel.gemini.ghost
+
+import java.awt.Shape
+import java.awt.geom.Ellipse2D
+
+import edu.gemini.spModel.inst.ScienceAreaGeometry
+import edu.gemini.spModel.obs.context.ObsContext
+import scalaz._
+import Scalaz._
+
+object GhostScienceAreaGeometry extends ScienceAreaGeometry {
+  private val size = 7.5 * 60 // arcmin
```

Comment:
Can this be defined with a richer type?

Reply:
When we re-do this for GPP we should be using an `Angle` or something but, to be consistent with everything else in the OT/TPE, I think this is probably fine here.

Reply:
I had thought about using `Angle`, but this is how we did it for GMOS and F2, so I just followed suit.